src/resources/views/province/show.blade.php

```diff
@@ -0,0 +1,82 @@
+@extends('appshell::layouts.private')
+
+@section('title')
+    {{ $province->name }}
+@stop
+
+@push('page-actions')
+    <x-appshell::standard-actions
+        :model="$province"
+        :name="$province->name"
+        :edit-url="route('appshell.province.edit', [$country, $province])"
+        :delete-url="route('appshell.province.destroy', [$country, $province])"
+    />
+@endpush
+
+@section('content')
+    <div class="row mb-3">
+        <div class="col">
+            <x-appshell::card-with-icon icon="plan" type="success">
+                {{ $province->name }}
+
+                <x-slot:subtitle>
+                    PROVINCE
+                </x-slot:subtitle>
+            </x-appshell::card-with-icon>
+        </div>
+
+        <div class="col">
+            <x-appshell::card-with-icon icon="password" type="info">
+                {{ $province->code }}
+
+                <x-slot:subtitle>
+                    CODE
+                </x-slot:subtitle>
+            </x-appshell::card-with-icon>
+        </div>
+
+        <div class="col">
+            <x-appshell::card-with-icon icon="tag">
+                {{ $province->type->label() }}
+
+                <x-slot:subtitle>
+                    TYPE
+                </x-slot:subtitle>
+            </x-appshell::card-with-icon>
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="col flex">
+            <x-appshell::card class="col-md-6">
+                <x-slot:title>Parent</x-slot:title>
+
+                @if ($province->parent)
+                    <a href="{{ route('appshell.province.show', [$country, $province->parent]) }}" class="d-block mb-3">
+                        <button class="btn btn-sm btn-secondary" type="button">
+                            {{ $province->parent->name }}
+                        </button>
+                    </a>
+                @else
+                    <p class="mb-3">No parent available.</p>
```

Comment:
please, always translate UI texts
```suggestion
                    <p class="mb-3">{{ __('No parent available.') }}</p>
```